CHANGELOG.md

```diff
@@ -4,9 +4,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
-## [Unreleased]
+## [0.3.1] - 21.08.2023
 
-### Added 
+### Changed 
+
+- Changing the application icon in order to enable distributing releases through TestFlight. - [#127](https://github.com/ios-course/ironfoudation-team-project/pull/198)
```

Comment:
check the number
```suggestion
- Changing the application icon in order to enable distributing releases through TestFlight. - [#198](https://github.com/ios-course/ironfoudation-team-project/pull/198)
```

Reply:
Fixed with 4d5305b

Reply:
I apologize for the revert commit. I was fixing what was accidentally merged Main into this branch.